ocaml/xapi-consts/api_errors.ml

```diff
@@ -1395,3 +1395,7 @@ let telemetry_next_collection_too_late =
 let illegal_in_fips_mode = add_error "ILLEGAL_IN_FIPS_MODE"
 
 let too_many_groups = "TOO_MANY_GROUPS"
```

Comment:
This is a bug in merging VM anti-affinity feature.
The error messages have to be added through `add_error` so that they could be retrieved easily by SDK generating code.